src/build-index.js

```diff
@@ -56,6 +56,15 @@ async function sleep(ms) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+// Log function that can be inserted in a "then" chain
+function dolog(msg) {
+  return arg => {
+    console.log(msg);
```

Comment:
no cute emoji?!?

Reply:
Not a single one. Browser-specs is serious business.

Reply:
😭